cumulusci/robotframework/CumulusCI.py

```diff
@@ -241,3 +243,8 @@ def _run_task(self, task_class, task_config):
     def debug(self):
         """Pauses execution and enters the Python debugger."""
         set_pdb_trace()
+
+    def set_test_elapsed_time(self, time_in_seconds):
+        """Analogous to Set Test Message: sets the elapsed time for a test in reports and dashboards"""
+        self.perf_listener.set_elapsed_time(float(time_in_seconds))
+        BuiltIn().set_test_message(f"Elapsed time set by test : {time_in_seconds}")
```

Comment:
Wouldn't it make more sense to call [Set Tags](http://robotframework.org/robotframework/latest/libraries/BuiltIn.html#Set%20Tags) here, rather than leave it up to the listener to set the tags? 

I think that would make this keyword a bit more self-documenting. Plus, the text string used for the tag would then be defined in one place (this file) rather than two (this file for the docstring, the listener for the actual tag).